python/README.md

```diff
@@ -0,0 +1,55 @@
+# PyFeast
```

Comment:
Could you change this to FEAST for Python, or something similar? There is already an [old binding](https://github.com/EESI/PyFeast) called PyFeast, but it doesn't share any code with this one, and the old PyFeast has been broken for a long time.